tools/docker-compose/supervisor.conf

```diff
@@ -110,14 +128,28 @@ autostart = true
 autorestart = true
 stopasgroup=true
 killasgroup=true
-stdout_events_enabled = true
-stderr_events_enabled = true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
 
 [eventlistener:superwatcher]
 command=stop-supervisor
 events=PROCESS_STATE_FATAL
 autorestart = true
-stderr_logfile=/dev/stdout
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[eventlistener:rsyslog-4xx-recovery]
+command=/awx_devel/tools/scripts/rsyslog-4xx-recovery
+buffer_size = 100
+events=PROCESS_LOG_STDERR
+priority=0
+autorestart=true
+stderr_logfile=/dev/stderr
```

Comment:
I noticed this eventlistener doesn't specify a stdout_logfile. Is that so that restarting rsyslog will be seen in the docker container output?

Reply:
stdout contain only that 
```
READY
RESULT 2\nOK
```
message I don't want it to spam a file

and stderr will go into the container output (which contain message about restarting service)